src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -4,28 +4,43 @@ namespace Workleap.OpenApi.MSBuild;
 
 public sealed class ValidateOpenApiTask : CancelableAsyncTask
 {
+    private const string CodeFirst = "CodeFirst";
+    private const string ContractFirst = "ContractFirst";
+    
+    /// <summary>
+    /// 2 supported mode:
+    ///     - CodeFirst: Generate the OpenAPI specification files from the code
+    ///     - ContractFirst: Will use the OpenAPI specification files provided
+    /// </summary>
+    [Microsoft.Build.Framework.Required]
```

Comment:
Why specifying the full type name here, and the rest of the code isn't like that?